### ISSUE TITLE
Cannot change timesheet period once in new month

### ISSUE THREAD

**Issue by goodconcepts** [odoo/odoo#4010]
Impacted versions:
ODOO Saas (odoo.com)

Steps to reproduce:
Human resources --> time tracking --> My current timesheet --> timesheet period

Current behavior:
timesheet period cannot be edited. I'm only 3 days into the new month and try to add working hours for the last two days of the previous month, but ODOO doesn't let me.

Expected behavior:
Allow timesheet hours to be addec/changed for previous periods when not submitted to manager or approved by manager yet.

**Comment by Yenthe666** [comment]
Hi @goodconcepts ,

Thank you for your bug report and our apoligies that we haven't had the time to look at this bug report. We do our best to handle all of them, but we sometimes miss some.
At this point Odoo 8 is no longer supported as Odoo only supports three versions (9.0, 10.0, 11.0 at the moment).
If you can still reproduce this issue on Odoo 9, Odoo 10 or Odoo 11 we encourage you to create a new bug report.

Regards,
Yenthe

**Comment by barry1955** [comment]
I have this issue in Version 10. 
I don't know why, but I cant seem to set the period os the timesheet to what we require.
At present if I set say 7th Dec as the start start, I get the 8th as the first date and then its almost impossible to get the correct period. I have had issue with nearly every timesheet. 

I set the current period (fortnightly) as I said to 7th Dec, but the first date I entered time for was the 8th.  I'm not sure if that caused the change (it shouldn't).
